cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -131,6 +131,22 @@ def credentials_collection(appliance):
     return appliance.collections.ansible_credentials
 
 
+@pytest.fixture(scope="function")
+def new_zone(appliance):
+    zone_collection = appliance.collections.zones
+    zone = zone_collection.create(
```

Comment:
**zone_collection** is used only once. I would use it directly rather than assigning to new variable.